CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ Usage:
 * Fix #2467: Add support for specifying imagePullSecrets via resource configuration
 * Fix #3220: ImageEnricher#mergeEnvVariables causes error for empty env
 * Fix #3228: Springboot 3.3.1 layertools output format breaks LayeredJarGenerator
+* Fix #3232: KubernetesResourceUtilTest refactor
```

Comment:
Could you please elaborate on this line a bit? CHangelog is written for the users to get an idea of what's new in this specific release.

Reply:
This is not adding any feature or change to production code in the project.
It's just a refactor fix, there's no changelog entry needed, it's better and less confusing once this is published as part of the release notes to remove this entry.